Source/DoxygenTranslator/src/PyDocConverter.cpp

```diff
@@ -0,0 +1,932 @@
+    translatedComment += '\n';
+
+    const string formulaIndent = indent.getFirstLineIndent();
+    translatedComment += formulaIndent;
+    translatedComment += ".. math::\n";
+
+    formulaNL = '\n';
+    formulaNL += formulaIndent;
+    formulaNL += m_indent;
+    translatedComment += formulaNL;
+  }
+
+  std::string formula;
+  handleTagVerbatim(tag, formula, arg);
+
+  // It is important to ensure that we have no spaces around the inline math
+  // contents, so strip them.
+  const size_t start = formula.find_first_not_of(" \t\n");
+  const size_t end = formula.find_last_not_of(" \t\n");
+  if (start != std::string::npos) {
+    for (size_t n = start; n <= end; n++) {
+      if (formula[n] == '\n') {
+        // New lines must be suppressed in inline maths and indented in the block ones.
+        if (!inlineFormula)
+          translatedComment += formulaNL;
+      } else {
+        // Just copy everything else.
+        translatedComment += formula[n];
+      }
+    }
+  }
+
+  if (inlineFormula) {
+    translatedComment += "`";
+  } else {
+    translatedComment += '\n';
+  }
+}
+
+void PyDocConverter::handleCode(DoxygenEntity &tag,
+                                std::string &translatedComment,
+                                const std::string &arg)
+{
+  IndentGuard indent(translatedComment, m_indent);
+
+  trimWhitespace(translatedComment);
+  translatedComment += '\n';
+
+  // Use the current indent for the code-block line itself.
+  string codeIndent = indent.getFirstLineIndent();
+  translatedComment += codeIndent;
+
+  // Go out on a limb and assume that examples in the C or C++ sources use C++.
+  // In the worst case, we'll highlight C code using C++ syntax which is not a
+  // big deal (TODO: handle Doxygen code command language argument).
+  translatedComment += ".. code-block:: c++\n\n";
+
+  // For now on, use extra indent level for all the subsequent lines.
+  codeIndent += m_indent;
+
+  std::string code;
+  handleTagVerbatim(tag, code, arg);
+
+  translatedComment += codeIndent;
+  for (size_t n = 0; n < code.length(); n++) {
+    if (code[n] == '\n') {
+      // Don't leave trailing white space, this results in PEP8 validation
+      // errors in Python code (which are performed by our own unit tests).
+      trimWhitespace(translatedComment);
+      translatedComment += '\n';
+
+      // Ensure that we indent all the lines by the code indent.
+      translatedComment += codeIndent;
+    } else {
+      // Just copy everything else.
+      translatedComment += code[n];
+    }
+  }
+
+  trimWhitespace(translatedComment);
+  if (*translatedComment.rbegin() != '\n')
+    translatedComment += '\n';
+}
+
+void PyDocConverter::handlePlainString(DoxygenEntity& tag,
+                                       std::string& translatedComment,
+                                       const std::string&)
+{
+  translatedComment += tag.data;
+  if (tag.data.size() && tag.data[tag.data.size() - 1] != ' ')
+    translatedComment += "";
```

Comment:
I don't know what it was supposed to do, but it seems to have been like this since the very beginning (see d9e8d140e76d6eaf498230efd66a9c7bce0955d8) and it passes the tests in the current state, so let me just remove these lines as they indeed don't do anything.